content/oci/oci.go

```diff
@@ -56,19 +56,13 @@ type Store struct {
 	//   - Default value: true.
 	AutoSaveIndex bool
 
-	// AutoGC controls if the OCI store will automatically clean newly produced
-	// dangling (unreferenced) blobs during Delete() operation. For example the
-	// blobs whose manifests have been deleted. Tagged manifests will not be
+	// AutoGC controls if the OCI store will automatically clean dangling
+	// (unreferenced) blobs created by the Delete() operation. This includes the
+	// unreferenced sucessor blobs and referrers. Tagged manifests will not be
 	// deleted.
```

Comment:
Consider the following wording to reduce ambiguity
```suggestion
	// (unreferenced) blobs created by the Delete() operation. This includes the
	// referrers and the unreferenced successor blobs of the deleted content.
	// Tagged manifests will not be deleted.
```